lib/doc_categories/docs_legacy_constraint.rb

```diff
@@ -2,6 +2,6 @@
 
 class ::DocCategories::DocsLegacyConstraint
   def matches?(_request)
-    ::DocCategories.legacyMode?
+    ::DocCategories.legacy_mode?
```

Comment:
Just did this at the same time because I noticed the non-standard for ruby camelCase